src/mars_patcher/data/schema.json

```diff
@@ -652,6 +687,27 @@
                 "Italian",
                 "Spanish"
             ]
+        },
+        "BlockLayer": {
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "type": "object",
+                "properties": {
+                    "X": {
+                        "$ref": "#/$defs/TypeU8",
+                        "description": "The X position in the room that should get edited."
+                    },
+                    "Y": {
+                        "$ref": "#/$defs/TypeU8",
+                        "description": "The Y position in the room that should get edited."
+                    },
+                    "Value": {
+                        "$ref": "#/$defs/TypeU8",
```

Comment:
This should be a U16

Reply:
See below.

Reply:
oh wait you referred to something else whoops

Reply:
So the description is a complicated way of saying "it's a block from the tileset"? Which means it's a U16

Reply:
Yeah I thought you were referring to the room id. Fixed.